test/Microsoft.ML.Tests/TrainerEstimators/TreeEstimators.cs

```diff
@@ -233,5 +237,150 @@ public void LightGbmMultiClassEstimator()
+            // Raw score.
+            var nativeResult1 = new double[_rowNumber * _classNumber];
+
+            // Get parameters used in ML.NET's LightGBM
+            var gbmParams = gbmTrainer.GetGbmParameters();
+
+            // Call LightGBM C-style APIs to do prediction.
+            string modelString = null;
+            using (var ch = (mlContext as IChannelProvider).Start("Training LightGBM..."))
+            using (var pch = (mlContext as IProgressChannelProvider).StartProgressChannel("Training LightGBM..."))
+            {
+                var host = (mlContext as IHostEnvironment).Register("Training LightGBM...");
+                var gbmNative = WrappedLightGbmTraining.Train(ch, pch, gbmParams, gbmDataSet, numIteration : numberOfTrainingIterations);
+
+                int nativeLength = 0;
+                unsafe
+                {
+                    fixed (float* data = dataMatrix)
+                    fixed (double* result0 = nativeResult0)
+                    fixed (double* result1 = nativeResult1)
+                    {
+                        WrappedLightGbmInterface.BoosterPredictForMat(gbmNative.Handle, (IntPtr)data, WrappedLightGbmInterface.CApiDType.Float32,
+                            _rowNumber, _columnNumber, 1, (int)WrappedLightGbmInterface.CApiPredictType.Normal, numberOfTrainingIterations, "", ref nativeLength, result0);
+                        WrappedLightGbmInterface.BoosterPredictForMat(gbmNative.Handle, (IntPtr)data, WrappedLightGbmInterface.CApiDType.Float32,
+                            _rowNumber, _columnNumber, 1, (int)WrappedLightGbmInterface.CApiPredictType.Raw, numberOfTrainingIterations, "", ref nativeLength, result1);
+                    }
+                    modelString = gbmNative.GetModelString();
+                }
+            }
+
+            //////////////////////////////////////////////////////////////////////////////////////////////
+            // Compare native LightGBM's and ML.NET's LightGBM results
+            //////////////////////////////////////////////////////////////////////////////////////////////
+
+            Assert.True(modelString != null);
+            float sigmoidScale = (float)(double)gbmParams["sigmoid"]; // Parameter passed to LightGBM
+            for (int i = 0; i < _rowNumber; ++i)
+            {
+                double sum = 0;
+                for (int j = 0; j < _classNumber; ++j)
+                {
+                    Assert.Equal(nativeResult0[j + i * _classNumber], predictions[i].Score[j], 6);
+                    sum += MathUtils.SigmoidSlow(sigmoidScale* (float)nativeResult1[j + i * _classNumber]);
+                }
+                for (int j = 0; j < _classNumber; ++j)
+                {
+                    double prob = MathUtils.SigmoidSlow(sigmoidScale * (float)nativeResult1[j + i * _classNumber]);
+                    Assert.Equal(prob / sum, predictions[i].Score[j], 6);
+                }
+            }
+
+        }
```

Comment:
>} [](start = 8, length = 1)

Even though we are not making baseline comparisons, please call `Done()` at the end of every test.